modules/update-conditional-updates.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * updating/understanding-upgrade-channels-releases.adoc
+
+[id="update-conditional-upgrade-path{context}"]
+= Updating to a conditional upgrade path
+
+You can update a conditional upgrade path using the OpenShift CLI (`oc`) 4.10 or later.
+
+.Procedure
+
+. You can perform an upgrade, by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to <version> <.>
```

Comment:
We cover this in `modules/update-upgrading-cli.adoc`.  Possibly worth scoping this module down to just the "...but what about when a conditional update is not recommended?" case?

Reply:
Hmm, I'm a bit confused by the web vs. `oc` breakdown and the patch vs. minor breakdown:

```console
$ git --no-pager grep modules/update-upgrading origin/main --
origin/main:updating/updating-cluster-cli.adoc:include::modules/update-upgrading-cli.adoc[leveloffset=+1]
origin/main:updating/updating-cluster-rhel-compute.adoc:include::modules/update-upgrading-web.adoc[leveloffset=+1]
origin/main:updating/updating-cluster-within-minor.adoc:include::modules/update-upgrading-web.adoc[leveloffset=+1]
origin/main:updating/updating-cluster.adoc:include::modules/update-upgrading-web.adoc[leveloffset=+1]
```

I would expect all combinations (web patch, `oc` patch, web minor, `oc` minor), because the steps that you take for a patch update all happen for minor updates too.  Not sure how much of that we want to involve ourselves in in this PR.